models/gas/ethereum/gas_ethereum_fees.sql

```diff
@@ -0,0 +1,58 @@
+{{ config(
+    alias = 'fees',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_time','tx_hash','tx_amount_native']
```

Comment:
Question to help my understanding: Why does the unique key consist of these fields?

Reply:
This was just chosen because I don't expect to have duplicate rows using these columns, did you have something else in mind ?

Reply:
Maybe I'm overlooking something, but couldn't `tx_hash` be sufficient?

Reply:
yeah, that definitely makes sense :)